src/main/js/controls/Graph/Graph.js

```diff
@@ -453,6 +453,18 @@ class Graph extends Construct {
             ) {
                 updateAxesDomain(this.config, this.content[position]);
             }
+            if (
+                this.config.showNoDataText &&
+                this.content.every((content) =>
+                    utils.isEmpty(content.config.values)
+                )
+            ) {
+                drawNoDataView(this.config, this.svg);
+                redrawDatelineContent(this.scale, this.config, this.svg);
```

Comment:
Why do we need to call this function `redrawDatelineContent`?

Reply:
Dateline needs to appear above the no data view. And when there is no data on the click of the panning button, then no data view will be drawn above the dateline and we need to draw it again.